src/module/item/entity.js

```diff
@@ -350,6 +350,7 @@ export default class OseItem extends Item {
       actor: this.actor,
       tokenId: token ? `${token.parent.id}.${token.id}` : null,
       item: this._source,
+      itemId: this._source.id,
```

Comment:
As @carl-ellis mentioned, I don't either get the inclusion of this here. Other than that, the PR looks fine to me!